### ISSUE TITLE
incredible incorrect rate in CRNN

### ISSUE THREAD

**Issue by haoyijiang** [HHTseng/video-classification#23]
The accuracy rate in my experiment is super low,about 1.02%. In the python file of check prediction, optimizer file is not used?

**Comment by UlrichLiechtenstein** [comment]
the 1.02% accuracy is in training or testing? Have you adjusted the default parameters? it's possible to have this accuracy if the learning rate is not proper. 

Also in the check prediction, the CRNN model parameters are reloaded purely for prediction purpose, not for retaining, so the optimizer will not be used. But of course, you may adjust it according to your need. Hope this answers your question.

**Comment by HHTseng** [comment]
Hi, the 1.02% accuracy is in training or testing? have you adjusted the default parameters? it's possible to have this accuracy if the learning rate is not proper.

Also in the check prediction, the CRNN model parameters are reloaded purely for prediction purpose, not for retraining, so the optimizer will not be used. But of course, you may adjust it according to your need. Hope this answers your question.
